tests/GridFieldEditableColumnsTest.php

```diff
@@ -47,10 +45,9 @@ public function testProvidesEditableFieldsInColumns()
         $record->setCanEdit(true);
         $column = $component->getColumnContent($grid, $record, 'Title');
 
-        $this->assertInstanceOf(DBHTMLText::class, $column);
         $this->assertMatchesRegularExpression(
             '/<input type="text" name="TestGridField\[GridFieldEditableColumns\]\[100\]\[Title\]" value="foo"[^>]*>/',
-            $column->getValue()
+            $column
```

Comment:
The value for the column comes from `forTemplate()` which is now typed as `string`.